### PR TITLE
Use getTreeMaxSiblings() instead of numChildren() in max siblings limit check

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Add treeMaxSiblings property to radix node and state node
- Use getTreeMaxSiblings() instead of numChildren() in max siblings limit check

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/1067

### CODE REVIEW

File: db/state-node.js [hunk 1]
```diff
@@ -69,6 +72,8 @@ class StateNode {
   // from this calculation, since their sizes can vary and affect the gas costs and
   // state proof hashes.
   // 4(isLeaf) + 132(proofHash) + 8(treeHeight) + 8(treeSize) + 8(treeBytes) = 160
+  // NOTE(platfowner): treeMaxSiblings is not included in the node bytes computation since
```
Comment (minsulee2) [inline db/state-node.js:75]: 👍

File: db/state-node.js [hunk 2]
```diff
@@ -468,7 +484,8 @@ class StateNode {
         proofHash: this.radixTree.getRootProofHash(),
         treeHeight: 1 + this.radixTree.getRootTreeHeight(),
         treeSize: 1 + this.radixTree.getRootTreeSize(),
-        treeBytes: nodeBytes + this.radixTree.getRootTreeBytes()
+        treeBytes: nodeBytes + this.radixTree.getRootTreeBytes(),
+        treeMaxSiblings: Math.max(this.numChildren(), this.radixTree.getRootTreeMaxSiblings())
```
Comment (minsulee2) [inline db/state-node.js:488]: Q) Why still compare those two of numChildren and maxSiblings?

Comment (minsulee2) [reply db/state-node.js:488]: Oh, I got the point! Thanks for the explain!!

**Review by minsulee2 — APPROVED** [review]
LGTM! Thanks for the nice work :)

**Comment by platfowner** [comment]
Thanks for the fast review! Now merging.. @minsulee2

**Pull request merged**